wiki/Forum/en.md

```diff
@@ -0,0 +1,18 @@
+---
+stub: true
+tags:
+  - forum
+---
+
+#Forum
+
+The forums are a place where the [osu!](/wiki/Disambiguation/osu!) community can discuss a variety of topics. 
+
+The [forum index](https://osu.ppy.sh/community/forums) is generalised under a hierarchy:
+Forum -> Category (e.g; osu!) -> Section (e.g; development) -> Subforums (e.g; completed projects) -> Individual threads
```

Comment:
what I wrote in the review was just a comment with my idea, it shouldn't be put as-is into the article